docs/index.txt

```diff
@@ -75,6 +76,12 @@ Issues & Help
 Learn how to report bugs, contribute to the library, and find
 more resources in the :ref:`laravel-issues-and-help` section.
 
+Feature Compatibility
+---------------------
+
+Learn about Laravel features that are supported by
+the {+odm-long+} in the :ref:`laravel-feature-compat` section.
```

Comment:
Suggestion:

No change necessary, but I think this version might work better because it avoids passive voice and reusing the long form, which was used in the first mention on this page.

```suggestion
Learn about Laravel features that {+odm-short+} supports in the :ref:`laravel-feature-compat` section.
```